modP2pImpl/src/org/aion/p2p/impl/comm/NodeMgr.java

```diff
@@ -312,13 +312,13 @@ public INode getRandom() {
             try {
                 return this.getActiveNode((Integer) keysArr[random.nextInt(keysArr.length)]);
             } catch (IllegalArgumentException e) {
-                p2pLOG.error("getRandom-IllegalArgumentException", e);
+                p2pLOG.error("getRandom-IllegalArgumentException.", e);
                 return null;
             } catch (NullPointerException e) {
-                p2pLOG.error("<getRandom-NullPointerException", e);
+                p2pLOG.error("<getRandom-NullPointerException.", e);
```

Comment:
perhaps all message in this class should be between `<` and `>`

Reply:
okay. will change it.